src/Symfony/Bundle/FrameworkBundle/Resources/config/lock.xml

```diff
@@ -26,7 +26,7 @@
 
         <service id="lock.strategy.majority" class="Symfony\Component\Lock\Strategy\ConsensusStrategy" />
 
-        <service id="lock.factory.abstract" class="Symfony\Component\Lock\Factory" abstract="true">
+        <service id="lock.factory.abstract" class="Symfony\Component\Lock\LockFactory" abstract="true">
```

Comment:
we also need to conflict with lock < 4.4 and bump in require-dev

Reply:
done